mongoquery/__init__.py

```diff
@@ -29,7 +29,10 @@ def _match(self, condition, entry):
                 for sub_operator, sub_condition in condition.items()
             )
         else:
-            if isinstance(entry, Sequence):
+            # strings are a subtype of Sequence, thus checking first
+            if isinstance(entry, string_type):
+                return condition == entry
+            elif isinstance(entry, Sequence):
```

Comment:
Absolutely, /me just wanted to make it really obvious, but since it's practically the same, I'll update it to a compound statement shortly.